proto/cdcpb.proto

```diff
@@ -75,12 +75,16 @@ message Event {
         uint64 commit_ts = 2;
         LogType type = 3;
 
+	// generation is for pipelined DML protocol. See kvrpcpb.FlushRequest.generation.
+	uint64 generation = 10;
```

Comment:
Maybe fix the indent?

Reply:
Indent is correct. The line is just wrapped.